src/psyclone/transformations.py

```diff
@@ -3879,3 +3912,12 @@ def apply(self, node, options=None):
 
             # Add the global variable in the call argument list
             node.arguments.append(globalvar.name)
```

Comment:
Not for this PR but should node.arguments be symbols rather than strings?

Reply:
Possibly although that then moves us up into the "PSy layer" PSyIR and I don't think we use a symbol table there yet...